packages/metapackage/package.json

```
@@ -21,7 +21,9 @@
   "files": [
     "lib/*.d.ts",
     "lib/*.js.map",
-    "lib/*.js"
+    "lib/*.js",
+    "style/index.css",
+    "style/index.js"
```

Those change should not be happening.

```suggestion
    "lib/*.js"
```

---

Thanks for catching it twice 😃 I reintroduced it with the rebase.